community-operators/radanalytics-spark/sparkclusteroperator.0.3.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,128 @@
Vdc6WIr62hl6THNBs/ZhJcJtkvyDGXyulxIIFs/D0sy/j2msWo6+3CzP7uzG1qwODg4MLK5WKk8vlIpv+G+YMqL+vaY6fMlQqaYnrQzwBa+x1z2MOUXN9QSoXHMcRDGw16ZHzdJYdhuGlowdw7bXXNtoLcK5MIGkJ0ry1Pb/I86mXXnqp5+jRoy2LFs5Fb8/U87b8fvTw02jKZ7FwwVx4nl/HsoMxtShva7UgDDE6UsTI6BjODo2iXKqgElQRBAHGxkoolsoolSoolSuIoxjVMEQQ1HADvkE2vWFGaw3XdbXjONp1PZXJZONMJlPJ5XIjTU1NQ21tbSdnzpx5tLm5uSyljBzHCaWUkeu6sV0NZXhKbcYvSSbAoj+NA2tGDzbiPGBMox2CF1hfCjqhOUjI+txgtXLdPAAfakmJlyZswaamptHVq1d/+9FHH/2PDz/6jN/d3YX582aMR3Lb3uOqwcT445E+AXdRrzIEo9Hakseqqxfikcd/jve95wbM6JuK1pYmvLHzUNPevXt7Fy9efJA+q+33k6qvZk6N7iOgkA6esiIfxBjUVhWIylrlui5/jbFEqYT4Y0sHAwBBEOgGuE5yWbZs2SVdAvBEOg34OXQfj/r2J9nIc/z48elKKW/OrH5EUWzBGVG3T5NOpi3bduH1LTuwZNE8dHd3IYpqrZTBk2dw6vRZnDo1hFOnz2J4ZAylYhlhFCG0UT6OFWKljdEGsEsl2tvbw1wuN5LL+4Hv+yXHcQLHcSq+7we+75c9z6t4nhfk8/lyLperNDU1lVtaWspNTU0hyVDx3jI3PPaFpmtj2Uiy+3xgUPq2fS/N0HvDDM6kyoQLiiLM0ZmLBH0nrVsJGedOoqur69TatWvve+KJJz5937celP/5jz6KrikdrP639T4oCyDxUF1TIIaB1mpci1CbhBBEYG9f71TcdMNyPPST5/DBf/8O9PRMwevb9sixsbF2IcRB26pTQghtQT1NasOUrVgUn9h9xkZ0Sv+hlFLWK2tWMRFGoC0D0BDgZ9l/mpyBFQhFGIaJgxwaGvqViAJvJwaQPoHTJYChiIHaEJCmqKe1Vv39/UdnzZq1+dHHn1v6s+deyuTzOeRzWWSyGeSyGeRy2VpdZAwGT55GqRRg11sH8OWvfAtDQ6MolYP0kkzD0+7W1tbRyxf0n21qajrZ3Nx8olAonGxubh7u7u4+3dHRUbbOpY6Aw6THSTk2IYqQIbOlFYkzY/1gwVJ2U9dATGUUZjyvnlA+kCHbYyjSfydL/RPmHovUyWtSzz0f0FunA5B2AIwByKM/b/saVibolL5A8vyZM2ceXr169cMvvPDC733t698Tf/KfPoZ8LsvWjKvxxSK8BUj1PwgUHMcMEtkx+/kWXDYTxVIF3/ruE7j2mitq25GjKKdqSBvfI5DIiZETIOO3t7Xl/RPHX1vNP2U7CGT0yW067+39VPLCdV0EQWB830ccx4nTVkohCALzq3YBLkUHAJYBUJ+Y0iFF0fGuu+7atWLFio9u2bJl1t69exdt27Zt5v79+xefOnNytjEmQ6+39XTU3t4euq4bQPilnt6+YqFQGG1raxubPn36cHt7e9n3/SCTyYw2NTWNZTKZMJfLhfl8PiJDJ5IKscGsoWv2uesm8OywSBqU43Pk9FyVQsjpNkVjzf4F77una26bVkpOy7U1pLARVTaIvIk2XYPH6hzBBXR6jGigjDSZ8dPx46k+Le5IOQhyRFoIgSVLlrxaKpW6XnvttdXf+8Fj+PAf3Abfc+vER63QmJUSG2cBJsw/M5FTQhfHkVi+dD5GR0v42fMvUxmZIXkv+mxsoQcJftBn1OxfMmxtR4ENaiPARBxSqAl+GLuHQFH2oJQyjuPA8zxDxm/bg8TJ0LlcDmfPnjW+71/yDuBcOoA8+kueBVrgRFPbxQIgrtba9Pf3B319fbur1er+crnsBkHghmHoV6tVl+SUwjD0YMUcSTaM6nE25CHZAhBOTXWoVUPcbPslabY2THLj4zW5je6KGb+0jkHymt2eHJLPzmNcHCWtMSjsySUZTmKYU1AsjU+yCRKvRErtN1UGkCPgY70TgEVmMCaF+psGWUryGHu94Y6Ay26xeYC0U9FsmCa+6qqrHh8eHp72/AuvzG9rbcat714Nz3OSll8d/Zddp3JA1+ECEy+ZjIfV11+BXbsPYV/xmAjD0FE1sj7pAFKtrhk+Rcat6Zy18wAU+Y39/rVF/7UxRtH2H/taZbEUrbU2+XxelUol7Xme0Vpr3/d1EARobm6GUirhFlzyTMCFCxcaCwSKBuCfSTXgDGsDGUuM0J7noSbRHSkA0nEcJwxDJaWUvu9HQgidzWa15Z47dqmHa9dgJws72BQcSYxruyTEADBWA9+xWQeNHgtjjGJpPPVfKR0U9oRIhExVzVslToG14pRVNdbpYRvWj6eesaSsgU2kkRNIGHU2rRS0LZkBUoJpzqVbb5zEQkj8ZDoAItX+qkvRRb0ah0mn+czwuQ6ATqH9yb+UIbDyxHBUPJPJhO94xzu+OTQ09LmHH1nfBWGw9qZrkM9nbOqvagavCPDTKaegzwtu5rIZdHdPwf6Dx42UsmIzloSLwhR8NavpSU2I/tUMJEz+pQjueZ6pVqtk3EYpZaIoUjQBGASBcRwHlUpF5/N5RFFkjDEYGRnRnudBKaXDMEQul9Dfzd/+7d+aS7IEOHPmjOns7BSTRP90FmDGsSBhPM9L+ObWkCKttbEDKSS/rLTWtEWYFjzQJam16TyqlVDKsGWajgVoCNEmAVLwYRVrbJJthSEKKKX+ZLiJKAOl/+RAUsBfYuSMoCUaCW2wtBws3ReNIjgD/QUD5xLSDkfpU9/DpGTmSRSP0zuN67KEdPqf6vQgpRSUpPysJuaOI3EumUwmuPXWW7/20EMP/cmPH17fPDBwHP/ufTdj6rSOxNjHiiUMj46he2qHVSIe3ytwIRfHsvBc1x2yCH4iAUZjvHQ/ywg0KQCTchB
```

Comment:
`WATCH_NAMESPACE` isn't working with `0.3.3`. In `0.3.3` it's called `WATCHED_NAMESPACE`. So we can either change it back to `WATCHED_NAMESPACE` and make it work with `0.3.3`, or I can cut a release for `0.3.4`, rename the csv manifest, change the version here and there and use the updated `"WATCH_NAMESPACE"`. Up to you..

Reply:
The latter would follow OLM conventions which would be preferred, but if changing it to `WATCHED_NAMESPACE` helps make the operator work sooner then that works for me.